en/TripDescriptor.md

```diff
@@ -0,0 +1,13 @@
+### TripDescriptor

```

Comment:
This section is inconsistent with what is in index.md

Reply:
As is, most of it belongs in the spec as it describes functionality.

Reply:
Revised in https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/9f4196f2d399d95eeae3f99eb8cf328efdc15dc8 and https://github.com/MobilityData/GTFS_Realtime_Best-Practices/pull/1/commits/c37aee730e0be644642edf6dfb0b398fe3cc4898 to update and make them match.